src/App.css

```diff
@@ -1 +1,11 @@
-
+header {
+   background-color: black !important;
+}
+.header__title {
+   display: flex;
+   align-items: center;
+}
+.header__title p {
+   color: #fff;
```

Comment:
I would recommend you to use this colours guide Material-UI advice us: https://material-ui.com/ru/customization/color/.
It's not mandatory, but it would be nice.